models/classes/menu/SectionVisibilityFilter.php

```diff
@@ -53,10 +58,29 @@ public function isVisible(string $section): bool
             }
         }
 
+        $userSettings = $this->getUserSettingsService()->getCurrentUserSettings();
+        if (
```

Comment:
Blocks are separate by blank lines in OAT php standards
```suggestion
        $userSettings = $this->getUserSettingsService()->getCurrentUserSettings();
        
        if (
```

Reply:
Another point, lets avoid getting $userSettings, unless the feature flag is enabled. Better to move this condition to inside the IF